iwyu_preprocessor.cc

```diff
@@ -1087,4 +1126,21 @@ bool IwyuPreprocessorInfo::ForwardDeclareIsInhibited(
       ContainsKey(*inhibited_forward_declares, normalized_symbol_name);
 }
 
+bool IwyuPreprocessorInfo::ForwardDeclareInKeepRange(
```

Comment:
You should be able to drop the `clang::` qualifier throughout this method. If it's necessary for some symbol, prefer to add a using declaration.